### PR TITLE
Adding pyrte_rrtmgp recipe

### PR DESCRIPTION
**Pull request by giancastro**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by giancastro** [comment]
@conda-forge-admin please rerender

**Comment by giancastro** [comment]
@conda-forge-admin please rerender

**Comment by giancastro** [comment]
@conda-forge-admin, please rerender

**Comment by martin-g** [comment]
@conda-forge-admin please rerender

**Comment by jaimergp** [comment]
staged-recipes doesn't need to be rerendered. You can skip things but also feel free to ignore those builds that do not matter. Once approved and merged, the resulting feedstock will only generate the config for the platforms that are not skipped.

File: recipes/pyrte_rrtmgp/meta.yaml
Comment (jaimergp) [inline recipes/pyrte_rrtmgp/meta.yaml:28]: Please just use `{{ compiler('<language>') }}` functions.

**Comment by RobertPincus** [comment]
I am willing to act as a maintainer of this recipe.

**Comment by giancastro** [comment]
@conda-forge/help-python, ready for review!

File: recipes/pyrte_rrtmgp/meta.yaml
Comment (xhochy) [inline recipes/pyrte_rrtmgp/meta.yaml:37]: This should be unpinned to use the globally pinned version.
```suggestion
    - numpy
```

File: recipes/pyrte_rrtmgp/meta.yaml
Comment (xhochy) [inline recipes/pyrte_rrtmgp/meta.yaml:28]: If required, this should be in host.
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**
